.github/workflows/docs.yml

```diff
@@ -7,6 +7,8 @@ on:
     workflows: [ "Build Docker image (docs)" ]
     types:
       - completed
+  pull_request:
+    branches: [main]
```

Comment:
Can we add a path filter just below branches? I can understand your justification for when the dowhy source changes, but unit test changes shouldn't fire this off.

```yml
branches: [main]
paths:
  - 'docs/**/*'
  - 'dowhy/**/*'
  - pyproject.toml
  - poetry.lock
  - .github/workflows/docs.yml
```

Reply:
Makes sense. Added.